enhancements/ingress/host-network-configurable-ports.md

```diff
@@ -182,7 +173,20 @@ This  proposal edits `IngressController.operator.openshift.io/v1`.
 
 ### Operational Aspects of API Extensions
 
-N/A.
+#### External load-balancer health checks setup
+
+An external load balancer can send HTTP probes to the ingress controller's StatsPort on a given node with the path /healthz/ready to determine whether the ingress controller is ready to receive traffic on the node. 
+Sending health check requests every 5 or 10 seconds, with a 5-second timeout and with a threshold of two successful or failed requests to become healthy or unhealthy, respectively, are well-tested values.
+For proper operation, the load balancer must not forward traffic to a node until /healthz/ready reports ready, and the load balancer must stop forwarding within a maximum of 45 seconds after /healthz/ready starts reporting not-ready. 
+
+#### Investigating issues
+
+Cluster administrators should watch out for the ingress controller status after updates:
+In case the router pod cannot be scheduled due to port unavailability (or any other scheduling concerns), the ingress controller's conditions will report the scheduling failure.
+In such a case, the cluster administrator might need to inspect other pods or daemonsets that use the same port. Router pod logs, OVN/SDN logs might be helpful to identify the root cause for scheduling failure.
+
+Cluster administrators might also encounter issues if an ingresscontroller with HostNetwork strategy is using a port from the nodePort range: if a nodeport service attempts to use the same port, the router pod will not be able to get scheduled correctly. The nodePort ingress controller's conditions should report the scheduling failure.
```

Comment:
If a nodeport service attempts to use a port that the ingresscontroller is already using, then the nodeport service may not work properly.  I know that OVN-Kubernetes will log the conflict in its pods; I'm not sure about openshift-sdn's failure mode.